src/main/java/seedu/address/logic/parser/UnmarkAttendanceByStudentCommandParser.java

```diff
@@ -15,33 +19,41 @@
  */
 public class UnmarkAttendanceByStudentCommandParser implements Parser<UnmarkAttendanceByStudentCommand> {
 
+    private final Logger logger = LogsCenter.getLogger(UnmarkAttendanceByStudentCommandParser.class);
+
     /**
      * Parses the given {@code String} of arguments in the context of the UnmarkAttendanceByStudentCommand
      * and returns a UnmarkAttendanceByStudentCommand object for execution.
      * @throws ParseException if the user input does not conform the expected format
      */
     public UnmarkAttendanceByStudentCommand parse(String args) throws ParseException {
+        logger.info("Starting to parse arguments");
         ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args, PREFIX_ATTENDANCE, PREFIX_TUTORIAL);
 
         Index index;
 
         try {
             index = ParserUtil.parseIndex(argMultimap.getPreamble());
         } catch (ParseException e) {
+            logger.warning(String.format(MESSAGE_LOGGER_FOR_EXCEPTION, UnmarkAttendanceByStudentCommandParser.class
```

Comment:
The exception at parseIndex is already logged, so the logger here will result in duplicate logging, maybe can remove the logger here?

Reply:
Okay! Thanks for pointing it out, will remove